_includes/footer.html

```diff
@@ -19,4 +19,4 @@
   </ul>
 </div>
 
-<div class="page__footer-copyright">&copy; {{ site.time | date: '%Y' }} {{ site.name | default: site.title }}. {{ site.data.ui-text[site.locale].powered_by | default: "Powered by" }} <a href="http://jekyllrb.com" rel="nofollow">Jekyll</a> &amp; <a href="https://mademistakes.com/work/minimal-mistakes-jekyll-theme/" rel="nofollow">Minimal Mistakes</a>.</div>
\ No newline at end of file
+<div class="page__footer-copyright">The contents of this page, unless otherwise specified, is licensed under the <a href="https://creativecommons.org/licenses/by-sa/4.0/" rel="nofollow">Creative Commons Attribution-ShareAlike 4.0 International License</a>. Hosted on <a href="https://github.com/citation-style-language/citation-style-language.github.io">GitHub Pages</a>, powered by <a href="http://jekyllrb.com" rel="nofollow">Jekyll</a> &amp; <a href="https://mademistakes.com/work/minimal-mistakes-jekyll-theme/" rel="nofollow">Minimal Mistakes</a>.</div>
```

Comment:
Again a tiny detail: I think it should be either "The content (...) is licensed" or "The contents (...) are licensed" :-)